connector_helpscout/models/helpscout_customer/importer.py

```diff
@@ -26,6 +26,15 @@ def email(self, record):
         except IndexError:
             return
 
+    @mapping
+    def full_name(self, record):
```

Comment:
Weird. When does this happen?

Reply:
The specific case that led to this was the helpscout postmaster, but I assumed if that got through, there may be others.